onnxruntime/python/tools/quantization/operators/activation.py

```diff
@@ -20,3 +21,49 @@ def quantize(self):
 
         quantized_value = self.quantizer.quantized_value_map[node.input[0]]
         self.quantizer.quantized_value_map[node.output[0]] = quantized_value
+
+    def quantize(self):
+        node = self.node
+        if (node.op_type == "Relu" or node.op_type == 'Clip'):
```

Comment:
>( [](start = 11, length = 1)

nit: guess '==' has higher precedence already.
[https://www.mathcs.emory.edu/~valerie/courses/fall10/155/resources/op_precedence.html](https://www.mathcs.emory.edu/~valerie/courses/fall10/155/resources/op_precedence.html)

Reply:
() is ok.

Reply:
habit.